internal/quota/evaluator/ironcore/onmetal.go

```diff
@@ -4,8 +4,8 @@
 package ironcore
```

Comment:
Can we rename this file?

Reply:
Yes, we can. The package name has already been updated, so renaming the file won’t cause any issues. I’ve also renamed the other two files that were using the older name.